### PR TITLE
GraphQL changes related to database changes for keplr login

### PR DESCRIPTION
**Pull request by wgwz**
## Description

Related issue: https://github.com/regen-network/regen-registry/issues/939
Related PR: https://github.com/regen-network/registry-server/pull/187 

This PR is the result of running `yarn graphql:codegen` in my local development environment, with the database changes that we are making in the registry-server PR linked above.

---

### Author Checklist

*All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues.*

I have...

- [x] provided a link to the relevant issue or specification
- [x] reviewed "Files changed" and left comments if necessary
- [ ] confirmed all CI checks have passed

### Reviewers Checklist

*All items are required. Please add a note if the item is not applicable and please **add
your handle next to the items reviewed if you only reviewed selected items**.*

I have...

- [ ] confirmed all author checklist items have been addressed
- [ ] reviewed code correctness and readability
- [ ] verified React components follow DRY principles 
- [ ] reviewed documentation is accurate
- [ ] reviewed tests
- [ ] manually tested (if applicable)

### CODE REVIEW

**Review by blushi — APPROVED** [review]

**Comment by blushi** [comment]
Let's not merge this before https://github.com/regen-network/registry-server/pull/187 is merged into master

File: web-registry/src/generated/graphql.tsx [hunk 1]
```diff
@@ -27,6 +27,78 @@ export type Scalars = {
...
+  cursor?: Maybe<Scalars['Cursor']>;
+  /** The `Address` at the end of the edge. */
+  node?: Maybe<Address>;
+  /** Reads and enables pagination through a set of `Party`. */
+  partiesByAddressId: PartiesConnection;
+};
+
+
+/** A `Address` edge in the connection, with data from `Party`. */
+export type AccountAddressesByPartyAccountIdAndAddressIdManyToManyEdgePartiesByAddressIdArgs = {
```
Comment (haveanicedavid) [inline web-registry/src/generated/graphql.tsx:92]: lol @ the length of this type name 😂

**Review by haveanicedavid — APPROVED** [review]

**Review by mhagel — APPROVED** [review]

**Comment by wgwz** [comment]
@Mergifyio backport redwood hambach

**Comment by haveanicedavid** [comment]
@Mergifyio backport redwood hambach

**Pull request merged**